### PR TITLE
dependabot: monthly updates of github actions

### PR DESCRIPTION
**Pull request by consideRatio**
This is one of several PRs opened in the juptyterhub github organization to systematically configure dependabot to update github actions on a monthly basis, for more information see https://github.com/jupyterhub/team-compass/issues/636.

### CODE REVIEW

**Review by Sheila-nk — APPROVED** [review]

**Comment by Sheila-nk** [comment]
LGTM! 🚀 🚀  
Thank you @consideRatio for opening this PR!

**Comment by consideRatio** [comment]
Thanks for reviewing this @Sheila-nk and for your warm encouraging spirit!!! To me it makes a big difference as a contributor, thank you!

Sometimes when I review and approve something, I ask for someone else to review it also before we get something merged - but its a compromise of course since we are time constrainted. If a change is breaking or changing behavior of the projects code or a larger feature being added, I typically request another person to review it as well before merge.

What do you think about following up this approval with a clarification on what you think makes sense to do before a merge, or going for a merge directly? There are [guidance on the "when to merge" section from the jupyterhub team compass](https://jupyterhub-team-compass.readthedocs.io/en/latest/practices/pull_requests.html#when-to-merge-pull-requests) describing something like that. The same guidance also suggests I could also merge this now with your approval, but not all PR authors has the permissions to do so, so I think its a good practice to avoid using such priviledge.

**Comment by Sheila-nk** [comment]
@consideRatio Oh yes, I apologize! I should have followed up the approval with a merge or a message for you to feel free to merge this PR.

File: .github/dependabot.yaml
Comment (Sheila-nk) [inline .github/dependabot.yaml:5]: I just noticed that we might need to make an edit to the link provided for dependabot's status and logs.

Comment (consideRatio) [reply .github/dependabot.yaml]: Ah nice catch!!

**Comment by Sheila-nk** [comment]
Thank you @consideRatio for your contribution to this repository ❤️ 
I will now merge this PR 🚀 🚀

**Pull request merged**
